platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/ConsensusImpl.java

```diff
@@ -1111,26 +1115,26 @@ private long parentRound(EventImpl x) {
     }
 
     /**
-     * The last event created by m that is an ancestor of x (function from SWIRLDS-TR-2020-01).
+     * The last event created by the member at index m that is an ancestor of x (function from SWIRLDS-TR-2020-01).
      * This has aggressive memoization: the first time it is called with a given x, it immediately calculates and stores
      * results for all m.
      * This result is memoized.
      *
      * @param x
      * 		the event being queried
      * @param m
-     * 		the member ID of the creator
-     * @return the last event created by m that is an ancestor of x, or null if none
+     * 		the index of the member ID of the creator
+     * @return the last event created by member at index m that is an ancestor of x, or null if none
      */
-    private EventImpl lastSee(EventImpl x, long m) {
+    private EventImpl lastSee(EventImpl x, int m) {
```

Comment:
I know this is old code, but maybe we can start adding this kind of stuff as we make changes.

```suggestion
    private EventImpl lastSee(@Nullable final EventImpl x, final int m) {
```